### DISCUSSION TITLE
Update MILESTONES within sig-release repo => align to timing master

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-tractusx/sig-release#325] [Q&A] [answered]
Dear All,
as PI planning is currently underway, I pledge to immediately update the available milestones as follows:
milestone 23.12 => keep naming and due date of 08-Dec-2023.
milestone 24.03 => keep naming but update due date to 08-Mar-2024.
milestone 24.06 => update naming to 24.05 and update due date to 24-May-2024.
All of that will the be in complete alignment to the timing masterplan of Consortia & Association.
Comments welcome,
Roland

**Comment by FaGru3n** [comment, marked as answer]
updated on https://github.com/eclipse-tractusx/sig-release/milestones
![image](https://github.com/eclipse-tractusx/sig-release/assets/121097161/1a4a400d-f40e-4c12-a9f2-db837b0bf556)

**Reply by ghost** [reply]
Perfect - thank You so much.
Discussion completed.
